server/lorax_server/models/types.py

```diff
@@ -124,3 +124,19 @@ def to_pb(self) -> generate_pb2.Generation:
             next_tokens=self.next_tokens.to_pb(),
             generated_text=self.generated_text.to_pb() if self.generated_text is not None else None,
         )
+
+@dataclass
+class FlashBatch(ABC):
```

Comment:
This seems a bit overloaded with the FlashBatchForCausalLM. Maybe this should be FlashBatchForEmbedding?